deeplink/src/main/java/com/lyft/deeplink/DeepLink.java

```diff
@@ -63,7 +63,7 @@ public static boolean launchLyftApp(Context context, DeepLinkParams deepLinkPara
     static String createDeepLinkString(DeepLinkParams deepLinkParams) {
         StringBuilder sb = new StringBuilder();
         sb.append("lyft://ridetype?id=");
-        sb.append(deepLinkParams.getRideType());
+        sb.append(deepLinkParams.getRideType().getRideTypeKey());
```

Comment:
This is a null pointer exception according to the current implementation of `DeepLinkParams.java`.

Reply:
https://github.com/lyft/lyft-android-sdk/pull/13/files#diff-6edc060b52e27034e818eed5d40e3371R42

Reply:
Fixed the NPE, take a look when you get the chance and let me know if it's good to go :+1: